### DISCUSSION TITLE
station API issues?

### DISCUSSION THREAD

**Discussion by n-a-t-e** [weather-gov/api#682] [General]
Seeing issues with station queries, I'm not getting data after 2023-12-11T16:00:00Z, regardless of station, eg:

https://api.weather.gov/stations/46267/observations?start=2023-12-11T17:00:00Z
https://api.weather.gov/stations/PBFW1/observations?start=2023-12-11T17:00:00Z

**Comment by n-a-t-e** [comment]
I found the email to send API issues to and they made a bug report

**Comment by TimPasquini** [comment]
I've been having a similar issue with KALB since yesterday, last update 2023-12-11T17:33:00+00:00
